In [ ]:
import pandas as pd
import numpy as np
from scipy import stats

# __Problem Statement :__  


_Sales of products in four different regions is tabulated for males and females. Find if male-female buyer rations are similar across regions_

## __1 - Business Problem__  
Is the male-female buyer rations are similar across regions

## __2 - Data description__

α == 0.05 (95% Confidence)

Y == Discrete  
X == Discrete  

Since there are more than 2 variable we will perform Chi-Square test

## __3 - Chi-Square Test__  

In [ ]:
df = pd.read_csv("/content/BuyerRatio.csv")

H0 == The male-female buyer rations are similar across regions  
H1 == The male-female buyer rations are not similar across regions

In [ ]:
stats.chi2_contingency([df["East"], df["West"], df["North"], df["South"]])

Chi2ContingencyResult(statistic=1.5959455386610577, pvalue=0.6603094907091882, dof=3, expected_freq=array([[  42.76531299,  442.23468701],
       [ 146.81287862, 1518.18712138],
       [ 131.11756787, 1355.88243213],
       [  72.30424052,  747.69575948]]))

P value of Chi-Square test  == 0.66 > α

## Thus HO is accepted.  
## The male-female buyer rations are similar across regions  